managed/services/backup/backup_service.go

```diff
@@ -378,14 +381,14 @@ func (s *Service) SwitchMongoPITR(ctx context.Context, serviceID string, enabled
 				"current service id: %s, service type: %s", serviceID, service.ServiceType)
 		}
 
-		agents, err := models.FindPMMAgentsForService(tx.Querier, serviceID)
+		serviceAgents, err := models.FindPMMAgentsForService(tx.Querier, serviceID)
```

Comment:
Why do you think `serviceAgent` name is better?

Reply:
There's an imported package named `agents`